### DISCUSSION TITLE
Integrazione Tapo P100

### DISCUSSION THREAD

**Discussion by petretiandrea** [indomus/forum#390] [HUB: Home Assistant]
Ho realizzato un integrazione per la presa smart Tapo P100, inoltre sono supportati anche i modelli di lampadina, come L530.

Se può tornare utile a qualcuno :) [https://github.com/petretiandrea/home-assistant-tapo-p100](https://github.com/petretiandrea/home-assistant-tapo-p100)

È possibile installare l'integrazione direttamente tramite Hacs.

**Comment by snakuzzo** [comment]
Grazie mille @petretiandrea 
Non ho modo di testare in prima persona, ma ci fa molto piacere che tu abbia condiviso il tuo lavoro :)

**Comment by indomus** [comment]
Ottimo grazie, provvediamo ad aggiornare il sito in merito!

**Reply by indomus** [reply]
[https://indomus.it/componenti/tp-link-tapo-p100/](https://indomus.it/componenti/tp-link-tapo-p100/)

**Reply by Scialla** [reply]
Tanta Roba! Complimenti!

**Comment by petretiandrea** [comment]
Salve a tutti volevo solo aggiornarvi sul fatto che l'integrazione è maturata parecchio e supporta vari device "tapo" di tp link. Qua trovate la lista completa dei device supportati e dei nuovi sensori che l'integrazione fornisce: https://github.com/petretiandrea/home-assistant-tapo-p100#features

**Reply by indomus** [reply]
per fortuna ci date una mano. grazie, aggiorniamo le guide
